contrib/win32/install/docker/Dockerfile-build

```diff
@@ -0,0 +1,65 @@
+
+# escape=`
+
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt in the project root for license information.
```

Comment:
We also have docker [based build](https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/openssh/vstsbuild.ps1).

This can be removed.